airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/streams.py

```diff
@@ -721,6 +721,65 @@ def next_page_token(self, response: requests.Response) -> Optional[Mapping[str,
         return response.json().get("before_cursor")
 
 
+class Triggers(SourceZendeskSupportCursorPaginationStream):
+    """Triggers stream: https://developer.zendesk.com/api-reference/ticketing/business-rules/triggers/#list-triggers"""
+
+    def next_page_token(self, response: requests.Response) -> Optional[Mapping[str, Any]]:
+        if self._ignore_pagination:
+            return None
+        next_page = self._parse_next_page_number(response)
+        return next_page if next_page else None
+    
+    def request_params(
+        self, stream_state: Mapping[str, Any] = None, next_page_token: Mapping[str, Any] = None, **kwargs
+    ) -> MutableMapping[str, Any]:
+        params = {"page": 1, "per_page": self.page_size, "sort_by": "asc"}
```

Comment:
I think the `sort_by` should be `updated_at ` and the  `sort_order ` field should be `asc`.
[documentation](https://developer.zendesk.com/api-reference/ticketing/business-rules/triggers/#list-triggers)

Reply:
Thanks @am6010 for raising. I think other streams using same functions need to be changed also. Will fix